### PR TITLE
use gSP instead of gSSP

### PR DESCRIPTION
**Pull request by mwickett**
Switch to `getStaticProps` on the homepage. We'll add revalidate as part of the full homepage buildout.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -13,31 +13,34 @@ export default function Home({ feed }) {
...
-            name='description'
-            content='Every covid number has a story which deserves to be shared'
+            name="description"
+            content="Every covid number has a story which deserves to be shared"
           />
-          <link rel='icon' href='/favicon.ico' />
+          <link rel="icon" href="/favicon.ico" />
         </Head>
         <div className={landing.background} />
         <main className={landing.main}>
```
Comment (mwickett) [inline pages/index.tsx:22]: I need to setup a format command for lint-staged/husky.

File: pages/index.tsx [hunk 2]
```diff
@@ -76,7 +79,7 @@ export default function Home({ feed }) {
   )
 }
 
-export async function getServerSideProps() {
+export async function getStaticProps() {
```
Comment (mwickett) [inline pages/index.tsx:82]: This is the only non-formatting change.

**Review by silent1mezzo — APPROVED** [review]
👍 LGTM, could throw in `revalidate: 10` to allow for newly moderated stories.

**Comment by mwickett** [comment]
Yeah, absolutely. Let's leave it to @davesteinberg as he'll be working on the homepage avec feed etc.

**Pull request merged**